src/app/components/app-header/app-header.component.ts

```diff
@@ -66,9 +70,13 @@ export class AppHeaderComponent extends BaseComponent implements OnInit {
         private router: Router,
         private cdr: ChangeDetectorRef,
         private navigationService: NavigationService,
+        private lineageGraphHeigthService: LineageGraphHeigthService,
     ) {
         super();
     }
+    ngAfterViewInit(): void {
```

Comment:
Blank line before the method, make it explicitly public

Reply:
Fixed